multi-int-store/src/main/java/com/connexta/multiintstore/controllers/StoreController.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.multiintstore.controllers;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestPart;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+@RestController
+@RequestMapping("/mis")
+public class StoreController {
+  @PutMapping(value = "/product/", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
```

Comment:
✏️ Should this have a `/` at the end?

Reply:
I think it should be removed.

Reply:
I removed it in the latest commit and added some tests.